buildtools/carthage/cartfile.go

```diff
@@ -0,0 +1,189 @@
+package carthage
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/fossas/fossa-cli/files"
+	"github.com/fossas/fossa-cli/log"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/rveen/ogdl"
```

Comment:
[nit] Third-party and first-party import groups should be separated by a newline. In general: standard library, third-party imports, first-party imports.